test/smoke/app_create_cli_test.go

```diff
@@ -0,0 +1,23 @@
+package smoke
+
+import (
+	"testing"
+
+	. "github.com/argoproj/argo-cd/test/e2e/fixture"
+
+	"github.com/argoproj/pkg/errors"
+	"gotest.tools/assert"
+)
+
+func TestCreateApp(t *testing.T) {
+
+	// Instructions on testing this locally:
```

Comment:
Can we make this test idempotent so it could be run repeatedly times? 

I suggest that each smoke test work in "smoke-test" project ( name can be parameterized using env variable  ) and the "smoke-test" project should be cleaned up before running each smoke test.